build.gradle

```diff
@@ -53,7 +57,7 @@ dependencies {
 	compile 'com.google.code.findbugs:findbugs:2.0.3'
 	compile 'com.sk89q:worldguard:6.1.1-SNAPSHOT'
 	compile 'net.milkbowl.vault:Vault:1.6.6'
-	compile 'net.sacredlabyrinth.Phaed:PreciousStones:10.0.5'
+	compile 'com.github.marcelo-mason:PreciousStones:24e3d4b'
```

Comment:
Wouldn't it be better if you use `1.12-SNAPSHOT` as version? So it gets the latest commit from that branch.

Reply:
Done, thanks